rapp_web_services/services/object_detection_clear_models/svc.js

```diff
@@ -0,0 +1,93 @@
+/***
+ * Copyright 2015 RAPP
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Authors: Konstantinos Panayiotou, Maciej Stefańczyk
+ * Contact: klpanagi@gmail.com, m.stefanczyk@elka.pw.edu.pl
+ *
+ */
+
+
+/**
+ * @fileOverview
+ *
+ * [Object Recognition - Clear Models] RAPP Platform front-end web service.
+ *
+ *  @author Maciej Stefańczyk
+ *  @copyright Rapp Project EU 2015
+ */
+
+
+var path = require('path');
+var Fs = require(path.join(ENV.PATHS.INCLUDE_DIR, 'common', 'fileUtils.js'));
+
+var interfaces = require(path.join(__dirname, 'iface_obj.js'));
+
+const rosSrvName = "/rapp/rapp_object_detection/clear_models";
+
+
+/**
+ *  [Object Recognition - Clear Models]
+ *  Handles requests to object_detection/clear_models RAPP Platform Service
+ *
+ *  Service Implementation.
+ */
+function svcImpl(req, resp, ros) {
+  var rosMsg = new interfaces.ros_req();
+  rosMsg.user = req.body.user;
```

Comment:
Retrieve `user` from req object. A `username` property is applied to the `req` object after successful authentication.

```js
var user = req.username
```